### ISSUE TITLE
Ghosts in menu 

### ISSUE THREAD

**Issue by robert00s** [koreader/koreader#4079]
* KOReader version: current version after #3983 (I'm not sure)
* Device: Kindle PW2, Kindle PW3

#### Issue
I have a lot of ghosts after closing widgets, switching top/bottom menu. Diagonal swipe (full refresh) causes the ghosts to disappear.
No problem with regular reading.
I think that this happens after #3983.

**Comment by NiLuJe** [comment]
Unless you disabled the option added in #3983, you get a full-flash when closing a menu, which basically solves this.

In-menu ghosting is another kettle of fish, and one where we decided doing nothing hurts less than doing something drastic.

**Comment by robert00s** [comment]
Thanks for explanation. I understand that now. 

> If that's truly annoying you, the toggle in the eInk section of the Screen setting is here for that.

What option do you mean exactly?

Is it possible to add option to use like before #3983 with full-flash. It would be nice to have that option but I don't know if it's a lot of work.

**Comment by NiLuJe** [comment]
It's the last option in the Screen menu (not in front of a device right
now).

**Comment by poire-z** [comment]
`Avoid mandatory black flashes in UI`: now enabled by default (for less ghosting) - to disable to get the old behaviour with less black flashes (and possibly more ghosting).

**Comment by NiLuJe** [comment]
Yep, that's it, thanks ;).

In which case, behavior is identical as before the PR.

**Comment by robert00s** [comment]
@poire-z Thanks. 
I use this option (very useful - after closing menu I have back flash). 
But I still have ghosts after switching in menu, tap submenu. Ghosts are also visible after closing OpenWithDialog or context menu in filemanager.
It seems to me that a few months ago there was no problem with that. Unless I see it only now :))
What about another devices like Kobo?

**Comment by NiLuJe** [comment]
If you see a black flash when closing a menu, then that option is disabled
(unchecked, the default), and you're seeing the new behavior.

Enabling it restores the previous behavior (verbatim, in this specific
case).

**Comment by robert00s** [comment]
Thanks.
